i18n/spanish.json

```diff
@@ -702,34 +702,34 @@
-    "cmd-blindtest-is-running" : "A blindtest is already running.. Do you want to stop it ? (yes/no)",
-    "cmd-blindtest-stop-blindtest" : "Stopping blindtest!",
-    "cmd-blindtest-invalid-param" : "The parameter {} isn't valid.",
-    "cmd-blindtest-invalid-value" : "The value {} isn't valid for the parameter {}.",
-    "cmd-blindtest-blindtest-info" : "Fetching information from MyAnimeList...",
-    "cmd-blindtest-fail" : "Fail",
-    "cmd-blindtest-start" : "Starting a blindtest game in this channel with {} songs!",
-    "cmd-blindtest-mal-fail" : "Failed to fetch information from MyAnimeList, try again later!",
-    "cmd-blindtest-timeout" : "Timeout",
+    "cmd-blindtest-no-enough-param" : "No se han dado suficientes parámetros para iniciar una prueba a ciegas ...",
+    "cmd-blindtest-mal-same-time" : "Solo puedes iniciar {} listas de anime al mismo tiempo.",
+    "cmd-blindtest-max-animelist" : "No puedes poner esta cantidad de listas de anime. (max es 5)",
+    "cmd-blindtest-empty" : "Esta lista de reproducción automática está vacía! Se requiere al menos una canción para iniciar una prueba a ciegas ...",
+    "cmd-blindtest-help" : "Inicia una prueba a ciegas en el canal actual.\n\nOpciones Disponibles:\n\nlongestword=true|false (Default to False):\nAceptar la palabra más larga como una buena respuesta.\n\nlistening=true|false (Default to False):\n No acepta respuestas, solo escucha las canciones.\n\ntimeout=number (Default to 120):\n Tiempo de espera antes de detener la canción si no se encuentra. Entre 3 y 300 segundos.\n\nseverity=number (Default to 100):\n Severidad en porcentaje de palabra para aceptar una respuesta. Entre 1 y 100%.\n\npercentage=number,number,number (Default to 100,0,0):\n Porcentaje de OP, ED, OST. La suma de todos los valores debe ser 100.\n\nwait=number (Default to 5):\n Tiempo para esperar antes de cada canción. Entre 0 y 30 segundos.\n\nargument=status (Default to all):\n Solo selecciona un tipo de anime visto en las listas de anime dadas.\nLos estados disponibles son: todos, completado, observación, retener, descartado, ptw.\n\nremove=status,status2 (Default to ptw):\n Ignora un tipo (o varios) de anime visto (separados por comas) en las listas de anime dadas.\nLos estados disponibles son: completado, observación, retención, descartado, ptw.",
```

Comment:
Do not translate the available status as they have to be written in english only